docs/sources/auth/jwt.md

```diff
@@ -0,0 +1,89 @@
++++
+title = "JWT Authentication"
+description = "Grafana JWT Authentication"
+keywords = ["grafana", "configuration", "documentation", "jwt", "jwk"]
+aliases = ["/docs/grafana/latest/installation/jwt/"]
+weight = 300
++++
+
+# JWT Authentication
+
+You can configure Grafana to accept a JWT token provided in the HTTP header. The token is verified using a public key(s) stored in a file or provided by the JWKS endpoint.
+You can specify some of the JSON fields embedded in JWT (called claims) to use either as a username or as an email to sign in.
+
+## Enable JWT
+
+In order to use JWT authentication you'll first need to enable JWT in the [main config file]({{< relref "../administration/configuration.md" >}}) as well as specify the header name that contains a token.
+
+```ini
+[auth.jwt]
+# By default, auth.jwt is disabled.
+enabled = true
+
+# HTTP header to look into to get a JWT token.
+header_name = X-JWT-Assertion
+```
+
+## Configure login claim
+
+To identify the user, some of the claims needs to be selected as a login info. You could specify a claim that contains either a username or an email of Grafana user.
+
+Typically, the subject claim called `"sub"` would be used as a login but it might also be set to some application specific claim.
+
+```ini
+# [auth.jwt]
+# ...
+
+# Specify a claim to use as a username to sign in 
```

Comment:
This line has no period. Others comment lines do. Please be consistent. We generally put periods at the end of sentences in code comments.